backend/drm/drm.c

```diff
@@ -1304,8 +1304,14 @@ void scan_drm_connectors(struct wlr_drm_backend *drm) {
 				wl_list_insert(&wlr_conn->output.modes, &mode->wlr_mode.link);
 			}
 
+			bool is_mst = false;
+			drmModePropertyBlobRes *blob = drmModeGetPropertyBlob(drm->fd, wlr_conn->props.path);
+			if (blob && blob->data && strstr(blob->data, "mst:"))
```

Comment:
Maybe we can check `strstr(…) == blob->data` to make sure it's a prefix?

Reply:
`strncmp` seems like it would be more appropriate.

Reply:
Switched to strncmp  since we are checking only the first 4 bytes anyway